_quickjs.pyi

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Callable, Literal, TypedDict
+
+
+def test() -> Literal[1]: ...
+
+JSValue = None | bool | int | float | str | Object
+
+class Object():
+    def json(self) -> str: ...
+    def __call__(self, *args: Any) -> Any: ...
+
+class MemoryDict(TypedDict):
```

Comment:
I think we can remove this. It seems a little excessive and hard to maintain. It is just a `Dict[str, int]`

Reply:
Fixed in 50ffc9c50ea2964751df63a37dcc2ded8613a024.